src/cablab/providers/albedo.py

```diff
@@ -33,15 +29,15 @@ def get_variable_descriptors(self):
         return {
             VAR_NAME_BRIGHT: {
                 'data_type': numpy.float32,
-                'fill_value': numpy.nan,
+                'fill_value': FILL_VALUE,
                 'units': '-',
```

Comment:
I think we discussed this before, it probably doesn't matter a lot, so we can just leave the FillValues as they come with the dataset.